TheMovieDB/TheMovieDBTests/MovieParserTest.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MovieParserTest.swift
+//  TheMovieDBTests
+//
+//  Created by Mariana Andrea Orman Berch on 7/10/21.
+//
+
+import XCTest
+@testable import TheMovieDB
+
+class MovieParserTest: XCTestCase {
+
+    func movieParserTest() throws {
```

Comment:
All the test methods must start with the word test or they won't be a test to run

Reply:
Fixed!